conf/report/report-template.html

```diff
@@ -33,7 +35,50 @@
           {% for tool in report %}
           { "orderDataType": "test-status" },
           {% endfor %}
-        ]
+        ],
+          initComplete: function() {
+            this.api().columns([2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17]).every(function() {
```

Comment:
what exactly does this do? do we have to have that list literal here?

Reply:
so I listed them out to not have the dropdown for the tags (first 2 columns) but I'll see if this can be done in some other way

Reply:
So this will have to be changed whenever we add a new tool ? This sounds brittle. Can this be formulated as 'everything but the first two columns' instead ?

Reply:
I believe this is resolved in the latest commit. I'll also look into adding a separate JS file for the script. Thanks for the suggestions!